code/modules/clothing/head/deck_helmets.dm

```diff
@@ -0,0 +1,49 @@
+/obj/item/clothing/head/deckhelmet
+	name = "deck helmet"
+	icon = 'icons/clothing/head/deckhelmet.dmi'
+	desc = "A plastic helmet with a visor, used in hangars on many ships."
+	icon_state = "deckcrew"
+	item_state = "deckcrew"
+	flags_inv = BLOCKHEADHAIR
+	contained_sprite = TRUE
+	armor = list(
+		melee = ARMOR_MELEE_SMALL,
+		bullet = ARMOR_BALLISTIC_MINOR,
+		laser = ARMOR_LASER_MINOR,
+		energy = ARMOR_ENERGY_MINOR,
+		bomb = ARMOR_BOMB_PADDED,
+		bio = ARMOR_BIO_MINOR,
+		rad = ARMOR_RAD_MINOR
```

Comment:
I dont think these -plastic- helmets should have any armor values, except ARMOR_MELEE_SMALL, if at all.

Reply:
fair point tbh, was thinking that myself